must-gather/collection-scripts/gather_lca

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="${1}"
+LCA_DATA_PATH="${BASE_COLLECTION_PATH}/lca-data"
+MUST_GATHER_LOGFILE_PATH="${2}"
+IBU_NS="openshift-lifecycle-agent"
+mkdir -p $LCA_DATA_PATH
+
+HOST_POD="$(oc -n ${IBU_NS} get pods -l app.kubernetes.io/name=lifecyle-agent-operator -o jsonpath='{.items[*].metadata.name}')"
+
+function get_ibu_crs() {
+    IBU_CRDS=($(oc get crd | awk '/lca.openshift.io/{print $1}'))
+
+    for CRD in "${IBU_CRDS[@]}"; do
+        echo "INFO: Collecting $CRD CR" | tee -a $MUST_GATHER_LOGFILE_PATH
+        oc adm inspect --dest-dir "${LCA_DATA_PATH}/cr" "${CRD}" -A
+    done
+}
+
+function get_ns_resources() {
+  echo "INFO: Collecting resources from ns ${IBU_NS}" | tee -a $MUST_GATHER_LOGFILE_PATH
+  oc adm inspect --dest-dir "${LCA_DATA_PATH}/ns-resources" "ns/${IBU_NS}"
+}
+
+function get_healthcheck_crs() {
+  echo "INFO: Collecting CRs used for healthcheck" | tee -a $MUST_GATHER_LOGFILE_PATH
+  healthcheck_dir=$LCA_DATA_PATH/cr-healthcheck
+  mkdir -p $healthcheck_dir
+  oc get Nodes -A -oyaml &> $healthcheck_dir/node.yaml
+  oc get ClusterOperator -A -oyaml &> $healthcheck_dir/cluster-operator.yaml
+  oc get MachineConfigPool -A -oyaml &> $healthcheck_dir/machine-config-pool.yaml
+  oc get ClusterVersion -A -oyaml &> $healthcheck_dir/cluster-version.yaml
+  oc get ClusterServiceVersion -A -oyaml &> $healthcheck_dir/cluster-service-version.yaml
```

Comment:
Should we include the SRIOV CRs, if SRIOV is installed?

Reply:
Updated to collect Sriov CRs